`System.InvalidOperationException: No service for type 'OctoshiftCLI.Contracts.ITargetGithubApiFactory' has been registered.` when running `gh bbs2gh grant-migrator-role`

## Description

`gh bbs2gh grant-migrator-role` does not seem to work, throwing a `System.InvalidOperationException: No service for type 'OctoshiftCLI.Contracts.ITargetGithubApiFactory'.

## Reproduction Steps

Run a simple, structurally valid `gh bbs2gh grant-migrator-role` command, e.g.:

```
gh bbs2gh grant-migrator-role --github-org caffeinesoftware --actor alan --actor-type USER --v
erbose
```

No HTTP requests are made, and you get an output like this:

```
[11:54] [INFO] You are running the latest version of the bbs2gh extension [v0.34]
[11:54] [ERROR] An unexpected error happened. Please see the logs for details.
```

with the following in the verbose log:

```
[11:54] [DEBUG] HTTP GET: https://raw.githubusercontent.com/github/gh-gei/main/LATEST-VERSION.txt
[11:54] [DEBUG] RESPONSE (OK): v0.34

[11:54] [INFO] You are running the latest version of the bbs2gh extension [v0.34]
[11:54] [ERROR] System.InvalidOperationException: No service for type 'OctoshiftCLI.Contracts.ITargetGithubApiFactory' has been registered.
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider , Type )
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider )
   at OctoshiftCLI.Commands.GrantMigratorRoleCommandBase.BuildHandler(GrantMigratorRoleCommandArgs args, IServiceProvider sp)
   at OctoshiftCLI.Extensions.CommandExtensions.<>c__DisplayClass1_0`3.<<ConfigureCommand>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Invocation.AnonymousCommandHandler.InvokeAsync(InvocationContext )
   at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseErrorReporting>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass12_0.<<UseHelp>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass19_0.<<UseTypoCorrections>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__18_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseParseDirective>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__5_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass8_0.<<UseExceptionHandler>b__0>d.MoveNext()
```

--- a/RELEASENOTES.md
+++ b/RELEASENOTES.md
@@ -1,1 +1,1 @@
-
+- Fix `gh bbs2gh grant-migrator-role` so it doesn't throw `System.InvalidOperationException`

--- a/src/bbs2gh/Program.cs
+++ b/src/bbs2gh/Program.cs
@@ -39,6 +39,7 @@ namespace OctoshiftCLI.BbsToGithub
                 .AddSingleton<DateTimeProvider>()
                 .AddSingleton<IVersionProvider, VersionChecker>(sp => sp.GetRequiredService<VersionChecker>())
                 .AddSingleton<BbsArchiveDownloaderFactory>()
+                .AddSingleton<ITargetGithubApiFactory>(sp => sp.GetRequiredService<GithubApiFactory>())
                 .AddHttpClient("Kerberos")
                 .ConfigurePrimaryHttpMessageHandler(() => new HttpClientHandler()
                 {

